### PR TITLE
Add USK

### PR DESCRIPTION
**Pull request by codehans**
This PR adds USK, referencing the image in this PR to chain-registry https://github.com/cosmos/chain-registry/pull/828

### CODE REVIEW

**Comment by silvermind** [comment]
@IbrarMakaveli upstream chain registy has been merged, 🥰

 if you merge this, will this fix the display issues uf this pool? https://frontier.osmosis.zone/pool/819
Or is there something else missing?

**Comment by thomasWos** [comment]
It will fix the display issue

**Comment by silvermind** [comment]
@thomasWos thx for your feedback. 💪 Happy to see when this get merged.

**Comment by JeremyParish69** [comment]
> @IbrarMakaveli upstream chain registry has been merged, 🥰
> 
> if you merge this, will this fix the display issues uf this pool? https://frontier.osmosis.zone/pool/819 Or is there something else missing?

No, it will not. The token must be added here: https://github.com/osmosis-labs/osmosis-frontend/tree/master/packages/web/config , to chain-infos.ts and ibc-assets.ts.
Taking a closer look at the addition of USK to the Chain Registry, I wonder if the slashes ('/') were meant to actually be colons (':')...

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -2240,6 +2240,33 @@
...
+        }
+      ],
+      "base": "ibc/44492EAB24B72E3FB59B9FA619A22337FB74F95D8808FE6BC78CC0E6C18DC2EC",
+      "name": "USK",
+      "display": "USK",
+      "symbol": "USK",
+      "ibc": {
+        "source_channel": "channel-3",
+        "dst_channel": "channel-259",
+        "source_denom": "factory:kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7:uusk"
```
Comment (sunnya97) [inline osmosis-1/osmosis-frontier.assetlist.json:2263]:
```suggestion
        "source_denom": "factory/kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7/uusk"
```

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 2]
```diff
@@ -2240,6 +2240,33 @@
...
+        }
+      ],
+      "base": "ibc/44492EAB24B72E3FB59B9FA619A22337FB74F95D8808FE6BC78CC0E6C18DC2EC",
+      "name": "USK",
+      "display": "USK",
+      "symbol": "USK",
+      "ibc": {
+        "source_channel": "channel-3",
+        "dst_channel": "channel-259",
+        "source_denom": "factory/kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7/uusk"
```
Comment (sunnya97) [inline osmosis-1/osmosis-frontier.assetlist.json:2263]:
```suggestion
        "source_denom": "factory:kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7:uusk"
```

Comment (sunnya97) [reply osmosis-1/osmosis-frontier.assetlist.json:2263]: Let's switch this back, now that I know the context

**Comment by silvermind** [comment]
❤️❤️

**Pull request merged**